requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Everything needed in production
 werkzeug<2.3
 Flask>=2.0
+flask[async]
```

Comment:
Hihi, I thought we could wait for Quart, but this looks ok enough now I guess. No conflicts detected with the other flask libraries?

Reply:
Nope, seems to work fine